config/armada/config.yaml

```diff
@@ -20,19 +20,12 @@ redis:
   password: ""
   db: 0
   poolSize: 1000
-eventsRedis:
-  addrs:
-    - redis:6379
-  password: ""
-  db: 0
-  poolSize: 1000
 eventsApiRedis:
   addrs:
     - redis:6379
   password: ""
   db: 1
   poolSize: 1000
-forceNewEvents: true
```

Comment:
Do we no longer need this flag?

Reply:
we do not- the old event store was populated from nats so will now be state- you *have* to use the new events